src/actions/index.js

```diff
@@ -390,18 +397,28 @@ function updateOutput(
      * }
      *
      */
+
+    const [outputComponentId, outputProp] = outputIdAndProp.split('.');
```

Comment:
I thought this was a mistake at first, but it looks like in the multi-output case it does not matter that these variables are incorrect. Perhaps only create these locals when `config.multi_output` for readability?